docs/content/en/docs/troubleshooting/troubleshooting.md

```diff
@@ -61,86 +61,79 @@ Remove cached credentials by running:
+
 ```
 Error: old cluster config file exists under my-cluster, please use a different clusterName to proceed
 ```
 The `my-cluster` directory already exists in the current directory.
 Either use a different cluster name or move the directory.
 
-### failed to create cluster: node(s) already exist for a cluster with the name
-```
-Performing provider setup and validations
-Creating new bootstrap cluster
-Error create bootstrapcluster	{"error": "error creating bootstrap cluster: error executing create cluster: ERROR: failed to create cluster: node(s) already exist for a cluster with the name \"cluster-name\"\n, try rerunning with --force-cleanup to force delete previously created bootstrap cluster"}
-Failed to create cluster	{"error": "error creating bootstrap cluster: error executing create cluster: ERROR: failed to create cluster: node(s) already exist for a cluster with the name \"cluster-name\"\n, try rerunning with --force-cleanup to force delete previously created bootstrap cluster"}ry rerunning with --force-cleanup to force delete previously created bootstrap cluster"}
-```
-A bootstrap cluster already exists with the same name. If you are sure the cluster is not being used, you may use the `--force-cleanup` option to `eksctl anywhere` to delete the cluster or you may delete the cluster with `kind delete cluster --name <cluster-name>`. If you do not have `kind` installed, you may use `docker stop` to stop the docker container running the KinD cluster.
-
-
 ### Memory or disk resource problem
-There are various disk and memory issues that can cause problems.
-Make sure docker is configured with enough memory.
-Make sure the system wide Docker memory configuration provides enough RAM for the bootstrap cluster.
-
-Make sure you do not have unneeded KinD clusters running `kind get clusters`.
-You may want to delete unneeded clusters with `kind delete cluster --name <cluster-name>`.
-If you do not have kind installed, you may install it from https://kind.sigs.k8s.io/ or use `docker ps` to see the KinD clusters and `docker stop` to stop the cluster.
- 
-Make sure you do not have any unneeded Docker containers running with `docker ps`.
-Terminate any unneeded Docker containers.
-   
-Make sure Docker isn't out of disk resources.
-If you don't have any other docker containers running you may want to run `docker system prune` to clean up disk space.
+
+There are various disk and memory issues on the admin machine that can cause problems. Make sure:
+* Docker is configured with enough memory.
+* The system wide Docker memory configuration provides enough RAM for the bootstrap cluster.
+* You do not have unneeded KinD clusters by running `kind get clusters`.
+    * You may want to delete unneeded clusters with `kind delete cluster --name <cluster-name>`. If you do not have `kind` installed, you may install it from https://kind.sigs.k8s.io/ or use `docker ps` to see the KinD clusters and `docker stop` to stop the cluster.
```

Comment:
```suggestion
    * You should delete unneeded clusters with `kind delete cluster --name <cluster-name>`. If you do not have `kind` installed, you can install it from https://kind.sigs.k8s.io/ or use `docker ps` to see the KinD clusters and run `docker stop` to stop the cluster.
```
Consider moving these two sentences so that are part of the main bullet and not a subbullet.